src/components/comparison/common.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components";
+import { Theme } from "../../utils/enums";
+import { getCurrentLayoutProperty } from "../../utils/layout";
+
+type LayoutProps = {
+  layout: string;
+};
+
+export enum Panels {
+  Layers,
+  ComparisonParams,
+}
+
+export const Container = styled.div<LayoutProps>`
+  display: flex;
+  flex-direction: column;
+  width: 359px;
+  background: ${({ theme }) => theme.colors.mainCanvasColor};
+  opacity: ${({ theme }) => (theme.name === Theme.Dark ? 0.9 : 1)};
+  border-radius: 8px;
+  padding-bottom: 26px;
+  position: relative;
+
+  max-height: ${getCurrentLayoutProperty({
+    desktop: "408px",
+    tablet: "408px",
+    mobile: "calc(50vh - 110px)",
+  })};
+`;
+
+export const PanelHeader = styled.div<{ panel: number }>`
+  display: flex;
+  align-items: center;
+  justify-content: ${(props) =>
+    props.panel === Panels.Layers ? "center" : "flex-start"};
+  background: transparent;
+  position: relative;
+  border-radius: 8px;
+  margin-top: 20px;
+  gap: 32px;
+`;
+
+export const CloseButton = styled.div`
```

Comment:
I made reusable `CloseButton` in https://github.com/visgl/loaders.gl-showcases/pull/101 . You need to await it to reuse it here

Reply:
Ok